[2023-01-28T13:46:43.265+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: exchange_rates.create_table_task manual__2023-01-28T13:46:42.013921+00:00 [queued]>
[2023-01-28T13:46:43.277+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: exchange_rates.create_table_task manual__2023-01-28T13:46:42.013921+00:00 [queued]>
[2023-01-28T13:46:43.278+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-28T13:46:43.279+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-28T13:46:43.280+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-28T13:46:43.297+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table_task> on 2023-01-28 13:46:42.013921+00:00
[2023-01-28T13:46:43.301+0000] {standard_task_runner.py:54} INFO - Started process 12830 to run task
[2023-01-28T13:46:43.304+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rates', 'create_table_task', 'manual__2023-01-28T13:46:42.013921+00:00', '--job-id', '13261', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates.py', '--cfg-path', '/tmp/tmp_qr3vxw1']
[2023-01-28T13:46:43.305+0000] {standard_task_runner.py:83} INFO - Job 13261: Subtask create_table_task
[2023-01-28T13:46:43.307+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/exchange_rates.py
[2023-01-28T13:46:43.411+0000] {taskmixin.py:205} WARNING - Dependency <Task(CurrencyScoopOperator): get_rate_AMD_RUB>, insert_rate_AMD_RUB already registered for DAG: exchange_rates
[2023-01-28T13:46:43.412+0000] {taskmixin.py:205} WARNING - Dependency <Task(PostgresOperator): insert_rate_AMD_RUB>, get_rate_AMD_RUB already registered for DAG: exchange_rates
[2023-01-28T13:46:43.413+0000] {dagbag.py:330} ERROR - Failed to import: /opt/***/dags/exchange_rates.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/exchange_rates.py", line 128, in <module>
    get_rate_task >> insert_rate >> wait_10_sec
NameError: name 'wait_10_sec' is not defined
[2023-01-28T13:46:43.420+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 13261 for task create_table_task (Dag 'exchange_rates' could not be found; either it does not exist or it failed to parse.; 12830)
[2023-01-28T13:46:43.436+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-01-28T13:46:43.460+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
